### DISCUSSION TITLE
Localize blueprint labels and instructions?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#6209] [Q&A] [answered]
I wonder how to localize blueprint labels and instructions in a multi-site setup. 

There is nothing in the docs about that ...

I tried to make subdirs like 'en" and 'de' - does not work.
I tried to use a {{ trans }} tag in the label field - does not work (only as content in fields itself).

Hm, there must be away, does it?

**Comment by jackmcdade** [comment, marked as answer]
There is no way at the moment – though it is an open feature request that will likely be worked in the near future.
